bigbluebutton-client/src/org/bigbluebutton/modules/chat/views/ChatToolbar.mxml

```diff
@@ -85,21 +110,36 @@
 		]]>
 	</mx:Script>
 
+	<common:TabIndexer id="tabIndexer" tabIndices="{[saveBtn, copyBtn, clearBtn]}"/>
+
 	<mx:Fade id="fadeOut" duration="200" alphaFrom="1.0" alphaTo="0.0" />
 	<mx:Fade id="fadeIn" duration="200" alphaFrom="0.0" alphaTo="1.0" />
 
-	<mx:Button id="saveBtn" label="{ResourceUtil.getInstance().getString('bbb.chat.saveBtn.label')}" 
-			   styleName="chatToolbarSaveButtonStyle"
+	<mx:Button id="saveBtn" 
+			   styleName="saveButtonStyle"
+			   width="22"
+			   height="22"
 			   toolTip="{ResourceUtil.getInstance().getString('bbb.chat.saveBtn.toolTip')}" 
 			   click="sendSaveEvent()"
-			   tabIndex="{baseIndex+1}"
-			   accessibilityName="{ResourceUtil.getInstance().getString('bbb.chat.copyBtn.accessibilityName')}"/>  
+			   accessibilityName="{ResourceUtil.getInstance().getString('bbb.chat.saveBtn.accessibilityName')}"/>  
 	
-	<mx:Button id="copyBtn" label="{ResourceUtil.getInstance().getString('bbb.chat.copyBtn.label')}" 
-			   styleName="chatToolbarCopyButtonStyle"
+	<mx:Button id="copyBtn" 
+			   styleName="copyButtonStyle"
+			   width="22"
+			   height="22"
 			   toolTip="{ResourceUtil.getInstance().getString('bbb.chat.copyBtn.toolTip')}" 
 			   click="sendCopyEvent()"
-			   tabIndex="{baseIndex+2}"
-			   accessibilityName="{ResourceUtil.getInstance().getString('bbb.chat.copyBtn.accessibilityName')}"/>  
+			   accessibilityName="{ResourceUtil.getInstance().getString('bbb.chat.copyBtn.accessibilityName')}"/> 
+	
+	<mx:Button id="clearBtn" 
+			   styleName="clearButtonStyle"
+			   width="22"
+			   height="22"
+			   visible = "{UsersUtil.amIModerator() &amp;&amp; publicChat}"
+			   enabled = "{UsersUtil.amIModerator() &amp;&amp; publicChat}"
+			   includeInLayout = "{UsersUtil.amIModerator() &amp;&amp; publicChat}"
```

Comment:
I needed both the user to be a moderator and the chat box to be public in order to have this button.